src/main/kotlin/eu/europa/ec/eudi/openid4vci/CredentialIssuerMetadataResolver.kt

```diff
@@ -17,13 +17,23 @@ package eu.europa.ec.eudi.openid4vci
 
 import com.nimbusds.jose.EncryptionMethod
 import com.nimbusds.jose.JWEAlgorithm
-import eu.europa.ec.eudi.openid4vci.CredentialResponseEncryption.NotRequired
+import eu.europa.ec.eudi.openid4vci.CredentialResponseEncryption.NotSupported
 import eu.europa.ec.eudi.openid4vci.internal.DefaultCredentialIssuerMetadataResolver
 import java.io.Serializable
 
 sealed interface CredentialResponseEncryption : Serializable {
```

Comment:
Here we need to introduce a data class containing the algs and ecryption method.
Then nest this to SupportedNotRequired and/or Required.